### PR TITLE
Add internal bay to case scraper

### PR DESCRIPTION
**Pull request by AdrianIlaga**
This PR is concerned with editing the Case scraper to get data on internal bays and save it as a hash

### CODE REVIEW

File: app/models/case.rb [hunk 1]
```diff
@@ -21,12 +22,12 @@ def self.parse_repo_page(response)
...
                             false
                           end
-    item[:bay_2_5] = get_int('Internal 2.5" Drive Bays')
-    item[:bay_3_5] = get_int('Internal 3.5" Drive Bays')
+    item[:internal_bays] = { 'bay2.5' => get_int('Internal 2.5" Drive Bays'), 'bay3.5' => get_int('Internal 3.5" Drive Bays') }
     item[:price] = response.xpath("//span[@class='price']").text.tr('^0-9.', '').to_f
     item[:name] = response.xpath("//div[contains(@class, 'product-info')]").css('h2').text
     item[:image] = response.xpath("//img[@class='nivo-main-image']").attr('src').value
     item[:motherboard_form_factor] = get_text('Motherboard Compatibility').split(',')
+    byebug
```
Comment (jarencudilla) [inline app/models/case.rb]: Remove this line.

**Comment by jarencudilla** [comment]
app/services/scraper.rb:12:5: C: Style/ClassVars: Replace class var @@class with a class instance var.
    @@class = model
    ^^^^^^^

what does the @@ do?

**Comment by AdrianIlaga** [comment]
> app/services/scraper.rb:12:5: C: Style/ClassVars: Replace class var @@Class with a class instance var.
> @@Class = model
> ^^^^^^^
> 
> what does the @@ do?

Makes it a global variable. It is necessary. Maam approved of it kanina

File: app/models/case.rb (hunk 1 above)
Comment (AdrianIlaga) [reply app/models/case.rb]: Removed byebug

**Review by EliezerAmbion — COMMENTED** [review]
We understand that it is a rubocop error. We've already talked with the team that we will just continue with the merge so that we could get new data.

**Review by DanielCadiz — COMMENTED** [review]
We've discussed this as a team, and will be proceeding to approve with the rspec errors.

**Comment by jarencudilla** [comment]
> > app/services/scraper.rb:12:5: C: Style/ClassVars: Replace class var @@Class with a class instance var.
> > @@Class = model
> > ^^^^^^^
> > what does the @@ do?
> 
> Makes it a global variable. It is necessary. Maam approved of it kanina

Thanks for clearing up my confusion.

**Pull request merged**